docs/user/masternodes/understanding.rst

```diff
@@ -68,13 +68,13 @@ means that Dash can scale more efficiently and deploy services more
 quickly than a blockchain run entirely by unpaid volunteers. The more
 masternodes, the better and safer the Dash network.
 
-As of November 2018, the Dash network has `over 5000 masternodes located
-<http://178.254.23.111/~pub/masternode_count.png>`_ in over `45
-countries <https://chainz.cryptoid.info/dash/masternodes.dws>`_ and
+As of November 2018, the Dash network has `almost 3500 masternodes located
```

Comment:
Please update the date here to match the statistics

Reply:
done